src/Common/src/Interop/Windows/Interop.Libraries.cs

```diff
@@ -9,73 +9,75 @@ internal static class Libraries
         internal const string Advapi32 = "advapi32.dll";
         internal const string BCrypt = "BCrypt.dll";
         internal const string Combase = "combase.dll";
-        internal const string Console_L1 = "api-ms-win-core-console-l1-1-0.dll";
-        internal const string Console_L2 = "api-ms-win-core-console-l2-1-0.dll";
-        internal const string CoreFile_L1 = "api-ms-win-core-file-l1-1-0.dll";
-        internal const string CoreFile_L1_2 = "api-ms-win-core-file-l1-2-0.dll";
-        internal const string CoreFile_L2 = "api-ms-win-core-file-l2-1-0.dll";
+        internal const string Console_L1 = "kernel32.dll";
+        internal const string Console_L2 = "kernel32.dll";
+        internal const string CoreFile_L1 = "kernel32.dll";
+        internal const string CoreFile_L1_2 = "kernel32.dll";
+        internal const string CoreFile_L2 = "kernel32.dll";
```

Comment:
Just curious: Should all these separate constants (e.g. `Console_L1`, `Console_L2`, `CoreFile_L1`, etc.) be deleted and references updated to use `Kernel32`, `User32`, etc., or is it beneficial to keep them separate?